src/ctia/http/middleware/auth.clj

```diff
@@ -3,22 +3,37 @@
             [compojure.api.meta :as meta]
             [ring.util.http-response :as http-response]))
 
-(defn wrap-authentication [handler]
+(defn add-id-to-request
+  "Given a request and a discoverd id and auth header
+  change the request accordingly"
```

Comment:
**Typo**: Given a request, a discover**E**d id and auth header, change the request accordingly.